packages/start/entry-client/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "solid-start/entry-client",
+  "module": "./dist/index.jsx",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "sideEffects": false
+}
```

Comment:
cf. https://github.com/solidjs/solid/blob/main/packages/solid/html/package.json

I'm not including `main` because I believe we require Node18.... I don't know nearly enough about Node to know if that's good enough justification but we're not using a "real" bundler like Rollup. Maybe later?